software/controller/lib/debug/commands.h

```diff
@@ -82,15 +82,38 @@ class PokeHandler : public MemoryHandler {
 //
 // Data passed to the command is a single byte which defines what the command
 // does:
-//  kFlushTrace - Used to disable the trace and flush the trace buffer
-//  kDownloadTrace - Used to read data from the buffer
+//  kFlush - Used to disable the trace and flush the trace buffer
+//  kDownload - Used to read data from the buffer
+//  kStart - Used to start recording in the trace buffer
+//  kGetVarId followed by var index (1 byte) - Used to get trace variables ID
+//  kSetVarId followed by var index (1 byte) and variable ID (2 bytes) - Used to
+//    set traced variable id
+//  kGetPeriod - Used to get the trace period
+//  kSetPeriod followed by desired trace period (4 bytes) - Used to set the
+//    trace period
+//  kCountSamples - Used to get the number of samples currently in the trace
+//    buffer
+
 class TraceHandler : public Handler {
 public:
   explicit TraceHandler(Trace *trace) : trace_(trace){};
   ErrorCode Process(Context *context) override;
 
+  enum class Subcommand {
```

Comment:
By making this a fixed with enum, I think we can get rid of many of the casts
```suggestion
  enum class Subcommand : uint8_t {
```